csu/inotify.c

```diff
@@ -0,0 +1,13 @@
+#include <stdint.h>
+
+// Qianxi Edit: add these two function because coreutils needs them but
```

Comment:
I'd tag this with BUG:

ie:

BUG: add these two function because coreutils needs them but... - Qianxi Chen